questionary/prompts/checkbox.py

```diff
@@ -246,6 +246,30 @@ def all(_event):
 
         perform_validation(get_selected_values())
 
+    @bindings.add(Keys.Home, eager=True)
+    def move_cursor_start(event):
+        ic.pointed_at = 0
+
+    @bindings.add(Keys.End, eager=True)
+    def move_cursor_end(event):
+        ic.pointed_at = len(ic.choices) - 1
+
+    @bindings.add(Keys.PageDown, eager=True)
+    def move_cursor_down_fast(event):
+        for _ in range(10):
```

Comment:
Does everything work correctly if there are fewer than 10 items? For example, if there are 9 items, page down should go to the 9th. Could you write a test for this?